From 738dfc1b33a83e9995ed7f65ec41ebe2fc6a90ed Mon Sep 17 00:00:00 2001
From: Ryan Eatmon <reatmon@ti.com>
Date: Wed, 1 Nov 2023 10:04:29 -0500
Subject: [PATCH] build: kbuild: Makefile.template: use SECONDARY

For some reason on master we are seeing an error:

| make[5]: *** .NOTINTERMEDIATE and .SECONDARY are mutually exclusive.  Stop.

Flip this back to .SECONDARY to fix this build issue.

Signed-off-by: Ryan Eatmon <reatmon@ti.com>

Upstream-Status: Pending

---
 build/linux/kbuild/Makefile.template | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/linux/kbuild/Makefile.template b/build/linux/kbuild/Makefile.template
index aa19d66..31e37fb 100644
--- a/build/linux/kbuild/Makefile.template
+++ b/build/linux/kbuild/Makefile.template
@@ -54,7 +54,7 @@
 
 include $(OUT)/config_kernel.mk
 
-.NOTINTERMEDIATE:
+.SECONDARY:
 
 define symlink-source-file
 @if [ ! -e $(dir $@) ]; then mkdir -p $(dir $@); fi
-- 
2.17.1

